pkg/plugin/state.go

```diff
@@ -695,6 +695,33 @@ func (e *AutoscaleEnforcer) handleUpdatedScalingBounds(vm *api.VmInfo, unqualifi
 	klog.Infof(fmtString, podName, pod.node.name, cpuVerdict, memVerdict)
 }
 
+func (e *AutoscaleEnforcer) handleNonAutoscalingUsageChange(vm *api.VmInfo, name util.NamespacedName) {
```

Comment:
```suggestion
func (e *AutoscaleEnforcer) handleNonAutoscalingUsageChange(vm *api.VmInfo, podName util.NamespacedName) {
```

It'd be good to make it clear that this refers to the name of the _pod_, not the _VM_